javaslang/src/main/java/javaslang/collection/TreeMultimap.java

```diff
@@ -65,7 +67,7 @@ private Builder(ContainerType containerType, SerializableSupplier<Traversable<?>
         }
 
         public <K extends Comparable<? super K>, V2 extends V> TreeMultimap<K, V2> ofEntries(Iterable<? extends Tuple2<? extends K, ? extends V2>> entries) {
-            return ofEntries((Comparator<? super K> & Serializable) K::compareTo, entries);
+            return ofEntries(naturalComparator(), entries);
```

Comment:
I think so - however, Java 9 isn't release and it is a Java 9 bug that was accepted (Prio 3). I hope it will be fixed soon - then it doesn't matter any more. However, the point is that we now use a Singleton instance instead of creating a lambda/method reference.